gr-pdu/python/pdu/qa_add_system_time.py

```diff
@@ -71,7 +71,7 @@ def test_002_timing(self):
                                         pmt.intern("systime"),
                                         pmt.from_double(0.0)))
         # should be sufficient tolerance
-        self.assertTrue(((t1 - t0) - 1) < 0.05)
+        self.assertAlmostEqual(t0, t1 - 1, delta=0.2)
```

Comment:
In addition to increasing the tolerance, I also changed the assertion to `assertAlmostEqual` so that the error message will display the two input values and the difference between them.

Reply:
Wow, still a lot of tolerance, but yeah, timing here is pretty uncertain...

Reply:
yea, these are benchmarking tools so I think this should catch most egregious errors. possibly this should be run many times and the mean error evaluated but this is an improvement.